### PR TITLE
PMM-12114 Bump grafana version to 9.2.18

### PR DESCRIPTION
**Pull request by matejkubinec**
PMM-12114

Link to the Feature Build: SUBMODULES-3272

Related PRs:
- [Grafana](https://github.com/percona-platform/grafana/pull/668)
- [Grafana Dashboards](https://github.com/percona/grafana-dashboards/pull/1504)

### CODE REVIEW

File: build/packages/rpm/server/SPECS/grafana.spec [hunk 1]
```diff
@@ -2,8 +2,8 @@
 %global commit          33423d34f211ce1ce5ce0a265a38f0709ec44360
 %global shortcommit     %(c=%{commit}; echo ${c:0:7})
 %define build_timestamp %(date -u +"%y%m%d%H%M")
-%define release         98
-%define grafana_version 9.2.13
+%define release         99
```
Comment (atymchuk) [inline build/packages/rpm/server/SPECS/grafana.spec:5]: @filipmikes1 I'm afraid we need to update the commit on line 2 as well :)

**Review by atymchuk — APPROVED** [review]

File: build/packages/rpm/server/SPECS/grafana.spec (hunk 1 above)
Comment (matejkubinec) [reply build/packages/rpm/server/SPECS/grafana.spec:5]: Hmm, right, where's that hash coming from?

Comment (atymchuk) [reply build/packages/rpm/server/SPECS/grafana.spec:5]: From the upstream repo. You can take it from the relevant [tag](https://github.com/grafana/grafana/releases/tag/v9.2.18), that appears be [2a6a04d0eb8bbe800eca86d8e7d3018a69a4daa6](https://github.com/grafana/grafana/commit/2a6a04d0eb8bbe800eca86d8e7d3018a69a4daa6).

Comment (atymchuk) [reply build/packages/rpm/server/SPECS/grafana.spec:5]: Sorry, I was wrong :| Take a commit hash from our own repo - https://github.com/percona-platform/grafana

**Review by idoqo — APPROVED** [review]

File: build/packages/rpm/server/SPECS/grafana.spec (hunk 1 above)
Comment (matejkubinec) [reply build/packages/rpm/server/SPECS/grafana.spec:5]: @atymchuk hash updated with squash commit

**Review by atymchuk — APPROVED** [review]

**Pull request merged**